app/components/billing-process.js

```diff
@@ -0,0 +1,53 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { equal, reads } from '@ember/object/computed';
+
+const STEPS = {
+  ONE: 'stepOne',
+  TWO: 'stepTwo',
+  THREE: 'stepThree'
+};
+
+export default Component.extend({
+  plan: service(),
+
+  showMonthly: reads('plan.showMonthly'),
+  displayedPlans: reads('plan.displayedPlans'),
+  selectedPlan: reads('plan.selectedPlan'),
+  showAnnual: reads('plan.showAnnual'),
+
+  scrollSection: null,
+  steps: [...Object.values(STEPS)],
```

Comment:
Values of complex types like Object and Array should not be assigned on the component factories directly, because that could lead to leaking of state (unexpected share of property changes between component instances). In this case it'd be better to make a computed property:
```suggestion
  steps: computed(() => [...Object.values(STEPS)]),
```
This property will be cached on first use and never update.